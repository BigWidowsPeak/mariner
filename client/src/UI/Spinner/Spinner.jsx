import React from 'react';
import classes from './Spinner.css';

const Spinner = (props) => (
  <div className={classes["sk-folding-cube"]}>
    <div className={[classes["sk-cube1"],classes["sk-cube"]].join(' ')}></div>
    <div className={[classes["sk-cube2"],classes["sk-cube"]].join(' ')}></div>
    <div className={[classes["sk-cube4"],classes["sk-cube"]].join(' ')}></div>
    <div className={[classes["sk-cube3"],classes["sk-cube"]].join(' ')}></div>
  </div>
);

export default Spinner;