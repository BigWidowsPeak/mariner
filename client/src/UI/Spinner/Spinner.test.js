import React from 'react';
import Spinner from './Spinner.jsx';
import renderer from 'react-test-renderer';

test('Spinner renders', () => {
  const component = renderer.create(<Spinner />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});