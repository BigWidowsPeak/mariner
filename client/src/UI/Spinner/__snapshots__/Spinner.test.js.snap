// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders 1`] = `
<div
  className="sk-folding-cube"
>
  <div
    className="sk-cube1 sk-cube"
  />
  <div
    className="sk-cube2 sk-cube"
  />
  <div
    className="sk-cube4 sk-cube"
  />
  <div
    className="sk-cube3 sk-cube"
  />
</div>
`;
